common/goods.txt

```diff
@@ -58,7 +58,7 @@ industrial_goods = {
 		cost = 2.5
 		color = { 186 50 40 }
 	}
-	artisinal_goods = {
+	exceptional_goods = {
```

Comment:
Make this precious_goods instead -- it'll fit the idea that all HPM-based mods follow